src/workerd/api/node/zlib-util.c++

```diff
@@ -624,6 +626,20 @@ void BrotliContext::setBuffers(kj::ArrayPtr<kj::byte> input,
   availOut = outputLength;
 }
 
+void BrotliContext::setInputBuffer(kj::ArrayPtr<kj::byte> input) {
+  nextIn = input.begin();
+  availIn = input.size();
+}
+
+void BrotliContext::setOutputBuffer(kj::ArrayPtr<kj::byte> output) {
+  nextOut = output.begin();
+  availOut = output.size();
+}
+
+kj::uint BrotliContext::getAvailOut() const {
+  return availOut;
+}
+
```

Comment:
These can be added to header file

Reply:
I didn't do this because the full setBuffers is defined just above, and I thought it was easier to read with them all in one place.